drivers/hmis/app/views/hmis_admin/project_groups/_form.haml

```diff
@@ -0,0 +1,23 @@
+= f.error_notification
+
+-# Project Group must have data source specified. Only allow selecting inclusion criteria for the given data source
+- ds = @project_group.data_source
+- organization_options = GrdaWarehouse::Hud::Organization.where(data_source_id: ds.id).options_for_select(user: current_user)
+
+%div.mb-4
+  %h3 Project Group Name
+  = f.input :name, required: true
+
+%div.mb-4
+  %h3 Inclusion Criteria
+  = f.simple_fields_for :inclusion_criteria do |ic|
+    = ic.input :project_ids, collection: [], as: :select_two, input_html: { multiple: true, data: { 'collection-path' => api_projects_path(selected_project_ids: @project_group.parsed_inclusion_criteria.project_ids, data_source_ids: [ds.id]) }, placeholder: 'Choose Projects'}, label: 'Projects', required: false
+    = ic.input :organization_ids, collection: organization_options, selected: @project_group.parsed_inclusion_criteria.organization_ids, as: :grouped_select_two, group_method: :last, input_html: {multiple: true, placeholder: 'Choose Organizations' }, label: 'Organizations', required: false
+    = ic.input :project_type_numbers, collection: Hmis::ProjectGroupCriteria.available_project_type_numbers, selected: @project_group.parsed_inclusion_criteria.project_type_numbers, input_html: { multiple: true, placeholder: 'Choose Project Types' }, label: 'Project Types', as: :select_two, required: false
+    = ic.input :all_projects_in_data_source, as: :boolean, label: 'Include all projects in data source', required: false
```

Comment:
Maybe put this first and toggle access to the others if checked?  (Again, happy for that to come later.)

Reply:
Yeah, for sure. I have a second branch in progress already for supporting additional criteria. I can try to incorporate this. That would need custom Javascript in the view, right? (muddling my way around rails views)

Reply:
Yeah, some basic custom JS in the template is probably how I'd roll it.  Alternatively, we could add the JS in `app/javascript` and then include it via the controller, but with something this small, I'd probably just add some wrapper divs with useful classes and put the toggle inline.